README.md

```diff
@@ -3,7 +3,8 @@
 Install tdp lib for dev:
 ```
 poetry install # Installs dependencies and the package in venv
-poetry run githooks setup
+poetry run pre-commit install
+poetry run pre-commit install --hook-type commit-msg
```

Comment:
Why a second command, the first one is not enough ?

Reply:
The first installs `pre-commit` hooks, the second install `commit-msg` hopks.

In `pre-commit` 2.18.0, you can configure (in the yaml), the hooks to install automatically, sadly, precommit 2.18.0 dropped support for python 3.6

Reply:
Ok I do not understand the `pre-commit` doc for `install` command, it is over complicated juste to install git hooks...

Reply:
I can change the doc to:
```
poetry run pre-commit install --hook-type pre-commit
poetry run pre-commit install --hook-type commit-msg
```

At least it will look the same